package.json

```diff
@@ -52,7 +52,7 @@
     "cz-conventional-changelog": "^3.3.0",
     "eslint": "^8.3.0",
     "execa": "^6.0.0",
-    "expect": "^1.20.2",
+    "expect": "^29.7.0",
```

Comment:
Can you change this to `"^28.1.3"` so that it can run on Node.js v12, that should fix [this error](https://github.com/birjj/svglint/actions/runs/6676161047/job/18161511215).

Reply:
There's another [problem](https://github.com/birjj/svglint/actions/runs/6692359102/job/18181695402?pr=92#step:5:9) with Node.js `v16.0.0`. I can change the version in workflow [`test.yml`](https://github.com/birjj/svglint/blob/33b56c2/.github/workflows/test.yml#L33) to `16.10.0`?

```
Run npm ci
npm ERR! code EBADENGINE
npm ERR! engine Unsupported engine
npm ERR! engine Not compatible with your version of node/npm: @jest/expect-utils@28.1.3
npm ERR! notsup Not compatible with your version of node/npm: @jest/expect-utils@28.1.3
npm ERR! notsup Required: {"node":"^12.13.0 || ^14.15.0 || ^16.10.0 || >=17.0.0"}
npm ERR! notsup Actual:   {"npm":"8.1.2","node":"v16.0.0"}

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/runner/.npm/_logs/2023-10-30T12_03_58_308Z-debug.log
Error: Process completed with exit code 1.
```

Reply:
Since this project aims to be [compatible with 16.0.0](https://github.com/birjj/svglint/blob/834abc486612349a9d4f30e8d7990506ddc0d96c/package.json#L60) I'd prefer to keep the CI test against that version. I checked once more and I believe `"^27.5.1"` should resolve the issue for Node.js v16.0.0 as well. From the [npm code explorer](https://www.npmjs.com/package/expect/v/27.5.1?activeTab=code) that version should work with `"node": "^10.13.0 || ^12.13.0 || ^14.15.0 || >=15.0.0"`.